oshi-core/src/main/java/oshi/software/os/mac/MacInstalledApps.java

```diff
@@ -78,4 +84,22 @@ private static ApplicationInfo createAppInfo(String name, Map<String, String> de
         return new ApplicationInfo(name, ParseUtil.getValueOrUnknown(details, "Version"), vendor, lastModifiedEpoch,
                 additionalInfo);
     }
+
+    private static String getMacDateTimeFormat(int style) {
+        CFTypeRef localeRef = CF.CFLocaleCopyCurrent();
+        try {
+            CFTypeRef formatter = CF.CFDateFormatterCreate(null, localeRef, style, style);
+            if (formatter == null) {
+                throw new IllegalStateException("CFDateFormatterCreate: null");
```

Comment:
While I copied this from the provided code, OSHI's pattern is to not throw exceptions.  Just return an empty string here.